firebase_admin/db.py

```diff
@@ -444,8 +444,8 @@ def start_at(self, start):
         Raises:
           ValueError: If the value is empty or None.
```

Comment:
If empty is ok, amend comments here and below

Reply:
Done. Good catch.